README.md

```diff
@@ -82,43 +104,143 @@ colcon build --symlink-install
 
 1. [Build](https://github.com/aws-lumberyard/ROSConDemo#download-and-install) and run the `ROSConDemo`
 2. Load level `Main`
+3. Start a simulation by hitting `CTRL+G`
+
+## Usage scenario
+
+### Single robot teleoperation with joystick
+1. Make sure that you have ROS2 workspace sourced
+
+```bash
+cd ~/o3de_kraken_ws
+source ./install/setup.bash
+```
+
+2. Make sure that you have `CycleDDS` as middleware 
+```bash
+export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+```
+
+3. Start teleoperation launch, make sure that you have your joystick plugged in
+```bash
+ros2 launch o3de_kraken_nav teleop.launch.py namespace:=apple_kraken_rusty_1
+```
+
+4. Spawn a single robot
+```
+ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'apple_kraken_rusty', xml: 'line1'}'
+```
+
+5. You should be able to drive the robot
+
+### Single robot navigation
+
+> Note: Slam is turned off by default since we have ground truth information about the robot's position from the simulator. However, it is possible to enable `slam_toolbox` forcefully. You can allow slam by adding `use_slam:=True` to the navigation launch command.
+
+
+1. Make sure that you have ROS2 workspace sourced
+
+```bash
+cd ~/o3de_kraken_ws
+source ./install/setup.bash
+```
+
+2. Make sure that you have `CycleDDS` as middleware 
+```bash
+export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+```
+
+3. Run the navigation stack
+
+```bash
+ros2 launch o3de_kraken_nav navigation_multi.launch.py namespace:=apple_kraken_rusty_1 rviz:=True
+```
+
+4. Spawn the robot
+```bash
+ros2 service call /spawn_entity gazebo_msgs/srv/SpawnEntity '{name: 'apple_kraken_rusty', xml: 'line1'}'
+```
 
-## Running nav stack for multiple vehicles
+5. You should be able to send goal to the robot. 
+Make sure that that you use only the first `2D Goal Pose`
+
+
+### Multiple robots navigation
 
 > Note: Slam is turned off by default since we have ground truth information about the robot's position from the simulator. However, it is possible to enable `slam_toolbox` forcefully. You can allow slam by adding `use_slam:=True` to the navigation launch command.
 
-1. For our scenario, spawn the following robots:
+1. Make sure that you have ROS2 workspace sourced
+
+```bash
+cd ~/o3de_kraken_ws
+source ./install/setup.bash
+```
+
+2. Make sure that you have `CycleDDS` as middleware 
```

Comment:
Move point 1 and point 2 to a separate section in usage scenario - the first one: base steps (for every scenario). No need for repetition here